pandas/tests/io/parser/c_parser_only.py

```diff
@@ -359,6 +359,15 @@ def test_parse_trim_buffers(self):
         # Check for data corruption if there was no segfault
         tm.assert_frame_equal(result, expected)
 
+        # This extra test was added to replicate the fault in gh-5291.
```

Comment:
worth making a different test for this?

Reply:
I think the whole point was to check whether or not the encoding argument would make a difference (notice that the calls are nearly the same).  That's why I added them as such.